ArticleTemplates/assets/js/bootstraps/common.js

```diff
@@ -45,6 +45,12 @@ function init() {
     initRichLinks();
     setupForms();
 
+    initSentry({
+        transport: Transports.FetchTransport,
+        dsn: 'https://8abc43d4e79b425eb6d4b5659ccd4020@sentry.io/40557',
```

Comment:
Which Sentry project does this refer to?

Reply:
The [webview](https://sentry.io/the-guardian/webview/) project that was used for apps 3/4 years ago. Still getting a lot of reports from old installs 😅

Reply:
Interesting, thanks @webb04!